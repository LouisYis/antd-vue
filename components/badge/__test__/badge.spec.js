import Vue from 'vue';
import Badge from '@/components/ui/badge';

const Constructor = Vue.extend(Badge);
const instance = new Constructor({
  propsData: {
    color: '#fff',
    type: 'error',
    size: 'small',
  },
}).$mount();

describe('Badge component', () => {
  test('component should match snapshot', () => {
    expect(instance.$el).toMatchSnapshot();
  });
});
