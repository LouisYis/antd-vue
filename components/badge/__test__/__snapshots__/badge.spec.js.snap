// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge component component should match snapshot 1`] = `
<span
  class="vk-badge is-error is-small"
  style="color: rgb(255, 255, 255);"
/>
`;
